integration_tests/mockApis/prisonApi.ts

```diff
@@ -50,7 +50,7 @@ const stubUserRoles = () =>
     response: {
       status: 200,
       headers: { 'Content-Type': 'application/json;charset=UTF-8' },
-      jsonBody: { dpsRoles: [], nomisRoles: [{ caseload: { id: 'LEI' }, roles: [] }] },
+      jsonBody: { dpsRoles: [{ code: 'ACTIVITY_HUB' }], nomisRoles: [{ caseload: { id: 'LEI' }, roles: [] }] },
     },
```

Comment:
This has reminded me that we will need another ticket to check that when a user tries to guess a URL, that they cannot access activities that are outside their own active caseload. Would you have a think about how that might fit into this authRole/authCaseload middleware check please?  Not for this ticket though.

Reply:
One for the API to manage? Return a 403 if it's not in their caseload?